mono/mini/mini-runtime.h

```diff
@@ -263,7 +263,9 @@ typedef struct MonoDebugOptions {
 	 */
 	gboolean top_runtime_invoke_unhandled;
 
+#ifdef ENABLE_NETCORE
```

Comment:
This could have been left alone imho..`#ifdef` really should be reduced/eliminated..

Reply:
@jaykrell this was actually what triggered the issue (I repro'd on the build machine). I don't really understand why though...

Reply:
The mere growth of the struct, or the setting it to true?
Understood, if the growth and I didn't debug it either..

Reply:
the mere growth